src/test/app/claims/claimStoreClient.ts

```diff
@@ -164,5 +168,73 @@ describe('ClaimStoreClient', () => {
         expect.fail() // Exception should have been thrown due to 500 response code
       })
     })
+
+    describe('Initiate citizen payment', async () => {
+      function resolveInitiatePayment () {
+        mock(`${claimStoreApiUrl}`)
+          .post(`/initiate-citizen-payment`)
+          .reply(HttpStatus.OK, paymentResponse)
+      }
+
+      it('should return nextUrl on successful initiate payment call', async () => {
+        resolveInitiatePayment()
+        const returnedUrl: string = await claimStoreClient.initiatePayment(claimDraft, claimant)
+        expect(returnedUrl).to.deep.equal(paymentResponse.nextUrl)
+      })
+
+      function mockInternalServerErrorOnInitiatePayment () {
+        mock(`${claimStoreApiUrl}`)
+          .post(`/initiate-citizen-payment`)
+          .times(retryAttempts)
+          .reply(HttpStatus.INTERNAL_SERVER_ERROR, 'An unexpected error occurred')
+      }
+
+      it('should propagate error responses', async () => {
+        mockInternalServerErrorOnInitiatePayment()
+        try {
+          await claimStoreClient.initiatePayment(claimDraft, claimant)
+        } catch (err) {
+          expect(err.statusCode).to.equal(HttpStatus.INTERNAL_SERVER_ERROR)
+          expect(err.error).to.equal('An unexpected error occurred')
+          return
+        }
+
+        expect.fail() // Exception should have been thrown due to 500 response code
```

Comment:
I find this clearer inside the `try` block - emphasises that the previous code bombed out, as expected